scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -13,19 +13,36 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Please remove TODO statements so that this is clean and can go into staging</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>